README.md

```diff
@@ -1 +1,63 @@
-# Global-Summit
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# TPPS Conference (Microverse captsone 1)
+
+> This website (purely fictional) is 
+- Homepage             
+![](images/capstone-desktop.png.png)
+- About Page
+![](images/capstone-about-desktop.png)
+- Mobile Homepage
+![](images/capstone-mobile.png)
+- Mobile About Page
+![](images/capstone-about-mobile.png)
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Linters
+
+## Live Demo
+
+[Live Demo Link]()
```

Comment:
- [ ] It looks like you have deployed the website but you forgot to share the live demo link here in the README file. Please add the live demo link so that visitors will easily interact with your website 😃